subprojects/build-cache-packaging/build-cache-packaging.gradle.kts

```diff
@@ -30,7 +30,6 @@ dependencies {
     implementation(project(":files"))
     implementation(project(":snapshots"))
 
-    implementation(library("jsr305"))
```

Comment:
❓ Shouldn't this project have a dependency to `:base-annotations` now?

Reply:
It will as soon as we remove `:base-services`.